client/src/pages/NeedHelp.js

```diff
@@ -127,9 +130,26 @@ const NeedHelp = withRouter((props) => {
     setState({ ...updatedAnswers });
     if (key === "location") {
       localStorage.setItem("needHelpAnswers", JSON.stringify(updatedAnswers));
```

Comment:
```suggestion
```

I don't think we need to use localStorage at all for onboarding anymore. To be honest I don't know why it was used in the beginning but it certainly was never working properly for this (see for example: #1765 I think it is related).